changhyun/app.js

```diff
@@ -0,0 +1,53 @@
+// 사용하는 npm 패키지(모듈)들 : express, nodemon, cors, dotenv, morgan
+
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const morgan = require('morgan'); //morgan 모듈 추가하기
+const dotenv = require('dotenv');
+
+const { DataSource } = require('typeorm'); //DataSource 모듈을 불러오기 
+
+dotenv.config(); //dotenv 사용
+// ES 모듈에서 환경 변수 불러오기(import) 다시 공부하기.
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+}) //process객체 : node.js 기본 글로벌 객체로 별도의 require 호출 없이 모든 모듈에서 접근이 가능 (process의 정확한 기능은 무엇인지?)
+
+myDataSource.initialize()
+  .then(() => {
+      console.log("Data Source has been initialized!")
+  })
+
+app.use(cors());
+app.use(morgan('combined')); //morgan 사용하기 (combined, common, dev, short, tiny)
+
+app.get('/ping', function (req, res, next){
+  res.json({message:'pong'});
+})
```

Comment:
`/ping` 이라는 엔드포인트로 구현되는 아주 간단한 API는 heath check (서버가 제대로 작동 중인지 최소한의 확인을 하기 위한) 용도로 사용 됩니다. 현재 의도한 바로는 request가 들어 올 때 pong이라는 메세지를 client에게 return 해주는 것인데 pong이 리턴이 됐다는 것은 API에 대한 요청이 성공적으로 이루어졌다는 것을 의미하고 이에 대한 적절한response code가 같이 반환 돼야 할 필요가 있습니다!

```suggestion
app.get("/ping", function (req, res, next) {
  return res.status(200).json({ message: "pong" });
});
```